plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -5094,14 +5112,18 @@ private void createEmptyTable(SchemaTableName schemaTableName, HiveStorageFormat
             LocationHandle locationHandle = locationService.forNewTable(transaction.getMetastore(), session, schemaName, tableName, Optional.empty());
             targetPath = locationService.getQueryWriteInfo(locationHandle).getTargetPath();
 
+            ImmutableMap.Builder<String, String> tableParamBuilder = new ImmutableMap.Builder();
+            tableParamBuilder.put(PRESTO_VERSION_NAME, TEST_SERVER_VERSION)
```

Comment:
```suggestion
            ImmutableMap.Builder<String, String> tableParamBuilder = ImmutableMap.<String,String>builder()
               .put(PRESTO_VERSION_NAME, TEST_SERVER_VERSION)
```

Reply:
I'll skip the <String,String> part as it is not necessary

Reply:
it's necessary when you `build() ..put().put()`.. Which is what we typically do.